src/provider.coffee

```diff
@@ -126,6 +127,9 @@ class Provider
     @context_label  = @body.context_label
     @context_title  = @body.context_title
 
+    # Load up the extensions!
+    extension.init(@) for extension_name, extension of extensions
```

Comment:
You are a lifesaver, I couldn't figure out what was going on for the life of me, thanks. I didn't even think to look in the object prototype. I'll finish up stuff when I get the time.